pkg/api/vlabs/const.go

```diff
@@ -84,14 +84,16 @@ const (
 const (
 	// Kubernetes153 is the string constant for Kubernetes 1.5.3
 	Kubernetes153 OrchestratorVersion = "1.5.3"
-	// Kubernetes157 is the string constant for Kubernetes 1.5.3
+	// Kubernetes157 is the string constant for Kubernetes 1.5.7
 	Kubernetes157 OrchestratorVersion = "1.5.7"
 	// Kubernetes160 is the string constant for Kubernetes 1.6.0
 	Kubernetes160 OrchestratorVersion = "1.6.0"
 	// Kubernetes162 is the string constant for Kubernetes 1.6.2
 	Kubernetes162 OrchestratorVersion = "1.6.2"
 	// Kubernetes166 is the string constant for Kubernetes 1.6.6
 	Kubernetes166 OrchestratorVersion = "1.6.6"
+	// Kubernetes170 is the string constant for Kubernetes 1.7.0
+	Kubernetes170 OrchestratorVersion = "1.7.0"
 	// KubernetesLatest is the string constant for latest Kubernetes version
 	KubernetesLatest OrchestratorVersion = Kubernetes166
```

Comment:
I previously did this but was asked not to so I did in a fixup.
This should be called `KubernetesDefault` instead.
With the go ahead from @JackQuincy I will rename that.